app/controllers/api/v2/providers_controller.rb

```diff
@@ -114,10 +133,29 @@ def update_enrichment
         enrichment.save
       end
 
+      def accredited_bodies_params
+        params
+          .fetch(:provider, {})
+          .except(
+            :train_with_us,
+            :train_with_disability,
+            :email,
+            :telephone,
+            :website,
+            :address1,
+            :address2,
+            :address3,
+            :address4,
+            :postcode,
+            :region_code
+          )
+          .permit(accredited_bodies: %i[provider_code provider_name description])
+      end
+
       def enrichment_params
         params
           .fetch(:provider, {})
-          .except
+          .except(:accredited_bodies)
```

Comment:
`accredited_bodies` is also being permitted below, shouldn't it be one or the other?